<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editListSubWindow</class>
 <widget class="QWidget" name="editListSubWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>editListSubWindow</string>
  </property>
  <widget class="QPushButton" name="editSaveButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>320</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确认编辑</string>
   </property>
  </widget>
  <widget class="QLabel" name="ISBNLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>53</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>书号</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="bookTextEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="priceLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>190</y>
     <width>53</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>价格</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="authorTextEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>150</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="bookLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>53</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>书名</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="priceTextEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>210</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="pressTextEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>270</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="authorLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>53</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>作者名</string>
   </property>
  </widget>
  <widget class="QLabel" name="pressLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>250</y>
     <width>53</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>出版社</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="ISBNTextEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>320</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>取消编辑</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
